### PR TITLE
fixes #272 - Added error info to CountryTweetMap

### LINKED ISSUE #272: Add error info and dark theme to map in CountryTweetMap app
Add error notifications for invalid input in CountryTweetMap app. Also add other themes to map so that users can switch between them.

### PR DESCRIPTION
**Pull request by djmgit**
Fixes issue #272, added error info and dark theme to map

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link: http://loklakapps.surge.sh/CountryTweetMap/
@kavithaenair @Achint08 @singhpratyush @SKrPl please review

File: CountryTweetMap/scripts/app.js [hunk 1]
```diff
@@ -49,6 +72,7 @@ app.controller("app", function ($scope, $http) {
         if (count !== undefined && count !== "") {
             url  += "&count=" + count;
         }
+        console.log(url);
```
Comment (vibhcool) [inline CountryTweetMap/scripts/app.js]: shall this be logged?

**Comment by vibhcool** [comment]
where could we find error info?

**Comment by Achint08** [comment]
@vibhcool Type an empty query and search for it. There will a snack bar appearing. Moreover, Try two consecutive queries one after the other.

**Comment by vibhcool** [comment]
I liked the dark theme, I suggest to apply this theme to complete app(with more colours) in subsequent PRs? :)

File: CountryTweetMap/scripts/app.js (hunk 1 above)
Comment (djmgit) [inline CountryTweetMap/scripts/app.js]: No it should not be :P , will fix it ASAP.

**Comment by djmgit** [comment]
@vibhcool yeah, I am working on it, somehow I need to attach event listener to the map so that I am able to apply dark css theme to the entire app. I will do it in subsequent PRs :)

**Comment by djmgit** [comment]
@vibhcool I have removed the log statement and also solved the date comparison issue which you mentioned :)

**Comment by djmgit** [comment]
@mariobehling got it, I will remove the dark theme. For allowing users to plugin apps into their own websites, app dependencies need to be in the same folder (as the app materials). Presently many of the app uses dependencies from elsewhere (like root folders). I will be working on it as soon as I am done with this app and another app.

**Comment by djmgit** [comment]
@mariobehling removed dark theme

**Pull request merged**